SlicerProstate/SlicerProstateUtils/mixins.py

```diff
@@ -246,13 +246,13 @@ def get3DEuclideanDistance(pos1, pos2):
     return distance3D
 
   @staticmethod
-  def dilateMask(label):
+  def dilateMask(label, dilateValue, erodeValue=0):
     imagedata = label.GetImageData()
     dilateErode = vtk.vtkImageDilateErode3D()
     dilateErode.SetInputData(imagedata)
-    dilateErode.SetDilateValue(1.0)
-    dilateErode.SetErodeValue(0.0)
-    dilateErode.SetKernelSize(12, 12, 1)
+    dilateErode.SetDilateValue(dilateValue)
+    dilateErode.SetErodeValue(erodeValue)
+    dilateErode.SetKernelSize(3, 3, 3)
```

Comment:
FYI, the segment editor module computes the kernel size from the pixel spacing:

https://github.com/Slicer/Slicer/blob/master/Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorMarginEffect.py#L61..L70

The kernel size must be an odd number, otherwise the output image may get shifted by one voxel.